README.md

```diff
@@ -150,6 +155,32 @@ The custom configuration options for the REST API are listed below:
   [Flask-SQLAlchemy configuration](https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/#configuration-keys)
   documentation.
 
+The custom configuration options for AMQP 1.0 messaging are listed below:
+
+* `IIB_MESSAGING_BATCH_STATE_DESTINATION` - the AMQP 1.0 destination to send the batch state change
+  messages. If this is not set, IIB will not send this type of message. If this is set,
+  `IIB_MESSAGING_URLS` must also be set.
+* `IIB_MESSAGING_BUILD_STATE_DESTINATION` - the AMQP 1.0 destination to send the build request state
+  change messages. If this is not set, IIB will not send this type of message. If this is set,
```

Comment:
`IIB will not send this type of message` => `IIB will not send this type of messages